AppleStore/ProductViewController.swift

```diff
@@ -0,0 +1,210 @@
+//
+//  ProductViewController.swift
+//  AppleStore
+//
+//  Created by Admin on 10.07.2021.
+//
+
+import UIKit
+
+final class ProductViewController: UIViewController {
+
+    //MARK: - Public properties
+    
+    private let productsScrollView: UIScrollView = {
+        let scrollView = UIScrollView()
+        scrollView.isPagingEnabled = true
+        scrollView.showsHorizontalScrollIndicator = true
+        return scrollView
+    }()
+    
+    public let firstProductImageView = UIImageView()
```

Comment:
```suggestion
    let firstProductImageView = UIImageView()
```
пока у нас нет других фреймворков мы не используем  public